app/templates/emailModal.html

```diff
@@ -68,7 +76,7 @@ <h6 id="emailLastSentSubject" class="text-center"></h6>
           </div> --> <!-- Commented out for demo purposes FIXME: add this option back and get it to work-->
           <br>
           <div class="modal-footer">
-            <button type="button" class="btn btn-secondary" data-bs-dismiss="modal" onclick="$('.modal').modal('hide')">Cancel</button>
+            <button type="button" id="cancel-btn" class="btn btn-secondary" data-bs-dismiss="modal">Cancel</button>
```

Comment:
We should be more specific with the ID of the button. Also, camelCase

Reply:
Bogus id name. I see that now. I shall make amends.